src/common/constants/array/index.ts

```diff
@@ -0,0 +1,3 @@
+const FIRST_ARRAY_IDX = 0
```

Comment:
Maybe better save constants for array in file "array.ts"? And not create separate folder. "src/common/constants/array.ts"

Reply:
> Maybe better save constants for array in file "array.ts"? And not create separate folder. "src/common/constants/array.ts"

Good point 👍🏻